src/addToClass.ts

```diff
@@ -16,7 +16,11 @@ function findStaticProperty(
   );
 }
 
-export default function addToClass(node: t.ClassDeclaration, state: ConvertState) {
+export default function addToClass(
+  node: t.ClassDeclaration,
+  state: ConvertState,
+  addVariables?: boolean
```

Comment:
Options are passed around in the `state` argument. Use that instead.

Reply:
You are right.